index.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
         var dotFolders            = this.readConfig('dotFolders');
         var serverSideEncryption  = this.readConfig('serverSideEncryption');
         var batchSize             = this.readConfig('batchSize');
+        var defaultMimeType       = this.readConfig('defaultMimeType');
```

Comment:
@achambers What do you think about adding the default as simply `application/octet-stream`? because I guess thats the only reasonable default I can think of. Have you got any other ideas?